install_base.sh

```diff
@@ -314,6 +314,8 @@ for arg in "$@"; do
         # gettext for autopoint
         pacman_packages+=(gettext autoconf libtool bison)
 
+        `git clone https://github.com/PierreARM/alpine-chroot-install -b pg/use_eval -C ${LISA_HOME}/tools`
```

Comment:
backquotes don't seem to be needed here (and in places where they have any use, they are better replaced by `$()` since 1. it nests correctly and 2. it's a bit less confusing when other quotes are in use